cpp/src/NVStrings.cu

```diff
@@ -472,10 +471,15 @@ int NVStrings::create_offsets( char* strs, int* offsets, unsigned char* nullbitm
     // copy memory to parameters (if necessary)
     if( !bdevmem )
     {
-        cudaMemcpy(offsets,d_offsets,(count+1)*sizeof(int),cudaMemcpyDeviceToHost);
-        cudaMemcpy(strs,d_strs,totalbytes,cudaMemcpyDeviceToHost);
+        cudaMemcpyAsync(offsets,d_offsets,(count+1)*sizeof(int),cudaMemcpyDeviceToHost);
```

Comment:
Since error checking is being improved, consider wrapping any CUDA calls like this in something like: https://github.com/rapidsai/cudf/blob/branch-0.9/cpp/src/utilities/error_utils.hpp#L173

Reply:
Done